src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
-body {
-  margin: 0;
+// body {
+//   margin: 0;
+// }
+@import "header.scss";
+@import "star.scss";
+@import "card.scss";
+@import "normalise.scss";
+// * {
+//   box-sizing: border-box;
+// }
+
+// html {
+//   font-family: "Roboto", sans-serif;
+// }
+
+// body,
+// img,
+// h2,
+// p,
+// header,
+// a,
+// nav,
+// ul,
+// li {
+//   margin: 0;
+// }
+
+.catalog__content {
+  display: flex;
+  flex-wrap: wrap;
+  width: 945px;
```

Comment:
catalog width should be calculated by formula `$catalog-width: 4 * $card-width + 3 * $card-margin;`